### PR TITLE
Adding doc strings for root_validator functions

### LINKED ISSUE #1044: Add doc strings to all @root_validator functions in the message parser
Some of the functions that we us to validate the bodies of requests to the message parser do not have documentation. Let's add doc strings to these functions.

### PR DESCRIPTION
**Pull request by robertandremitchell**
# PULL REQUEST

## Summary
Adding doc strings to provide more context to functions.

## Related Issue
Fixes #1044 

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

**Comment by robertandremitchell** [comment]
sorry, lot of issues getting the code to update/lint properly, but ready for review @bamader

File: containers/message-parser/app/models.py [hunk 1]
```diff
@@ -12,8 +12,14 @@
...
 
-# TODO: Add docstring here as part of separate ticket work
 def validate_secondary_reference_fields(values):
+    """
+    Function that determines whether the schema
+    given to message_parser is correctly formatted
+    Input: values (dict): This should be a schema that
+    should be parsed
+    Output: values, if parsing_schema correctly formmated
+    """
```
Comment (DanPaseltiner) [inline containers/message-parser/app/models.py:22]: Could you use the `:param:` and `:return:` nomenclature and format we have used elsewhere in the codebase for consistency?

Comment (robertandremitchell) [reply containers/message-parser/app/models.py:22]: thanks, @DanPaseltiner , updated! I tried to match the guidance here: https://github.com/CDCgov/phdi/blob/main/CONTRIBUTING.md

File: containers/message-parser/app/models.py [hunk 2]
```diff
@@ -12,8 +12,14 @@
...
 
 
-# TODO: Add docstring here as part of separate ticket work
 def validate_secondary_reference_fields(values):
+    """
+    Function that determines whether the schema
+    given to message_parser is correctly formatted
+    Input: values (dict): This should be a schema that
+    should be parsed
+    Output: values, if parsing_schema correctly formmated
```
Comment (bamader) [inline containers/message-parser/app/models.py:21]: I think we've got a misspelling on `formatted`

**Review by bamader — COMMENTED** [review]
Docstrings themselves look fine to me, the only outstanding changes I'd like to see are some super minor formatting things. The `validate_secondary_reference_fields` signature still needs to have the `:param:` and `:return:` added to it in place of `Input` and `Output`. And then for each of the other functions, if a line of the docstring runs over onto the following line, we just start that line indented two spaces rather than push everything to where the parameter started. For example:

```
:param bundle: The incoming FHIR bundle that has some properties and I'm rambling
  so you can see what I mean about moving down to another line
```

File: containers/message-parser/app/models.py (hunk 2 above)
Comment (robertandremitchell) [reply containers/message-parser/app/models.py]: thanks, @bamader ! updated

**Review by DanPaseltiner — APPROVED** [review]
LGTM. Thanks for re-formatting a bit.

**Review by bamader — APPROVED** [review]
LGTM, thanks for working through those formatting nits!

**Pull request merged**